### PR TITLE
[Rspec] author_followers_controller

### PR DESCRIPTION
**Pull request by nmtri2810**
## Related Tickets
- [ticket redmine](https://edu-redmine.sun-asterisk.vn/issues/71934)

## WHAT 
- Viết unit test cho author_followers_controller

## Evidence (Screenshot or Video)
image

### CODE REVIEW

**Comment by nmtri2810** [comment]
ready

**Comment by nmtri2810** [comment]
ready

File: spec/requests/author_followers_spec.rb [hunk 1]
```diff
@@ -1,18 +1,57 @@
...
+  before(:each) do
+    allow(controller).to receive(:authenticate_account!).and_return(true)
+    allow(controller).to receive(:current_account).and_return(account)
+  end
+
+  shared_examples "displays flash message" do |flash_type, message|
+    it "displays flash #{flash_type}" do
+      expect(flash.now[flash_type]).to eq I18n.t(message)
     end
   end
```
Comment (huongnt-2545) [inline spec/requests/author_followers_spec.rb:16]: em define ra file riêng để dùng chung cho controller khác nhé

**Comment by nmtri2810** [comment]
ready

**Comment by nmtri2810** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]

**Pull request merged**